spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -30,13 +32,16 @@
   describe 'user has created account' do
```

Comment:
use context here and put `sign_in` in the before block?  (I know its not directly related to this PR but ...